third_party/xla/xla/service/gpu/BUILD

```diff
@@ -3573,6 +3573,7 @@ xla_test(
     name = "gpu_compiler_test",
     srcs = if_gpu_is_configured(["gpu_compiler_test.cc"]),
     backends = ["gpu"],
+    tags = ["no_rocm"],
```

Comment:
I think it's better we need to disable that failed subtest (path incorrect due to xla migration from tensorflow) only instead of the whole test.

Reply:
I will do that.